great_expectations/exceptions/exceptions.py

```diff
@@ -48,11 +48,28 @@ class ExpectationSuiteError(DataContextError):
     pass
 
 
-class ExpectationSuiteNotAddedToStoreError(DataContextError):
-    def __init__(self) -> None:
+class ExpectationSuiteNotSavedError(DataContextError):
+    pass
+
+
+class ExpectationSuiteNotAddedToStoreError(ExpectationSuiteError):
+    def __init__(self, name: str) -> None:
+        super().__init__(
+            f"ExpectationSuite '{name}' must be added to the DataContext store before it can be saved. "  # noqa: E501
```

Comment:
Let's keep "store" out of this because that refers to an implementation detail that is private. I also prefer `updated` to `saved` in this context to make the add/update distinction clear. Change the error message to:
```
f"ExpectationSuite '{name}' must be added to the DataContext before it can be updated. 
```